src/main/g8/build.sbt

```diff
@@ -1,4 +1,4 @@
-scalaVersion := "3.3.1"
+scalaVersion := "3.4.1"
```

Comment:
I think you should use the latest LTS version. `3.3.3` I think.

Reply:
Changed to use current Scala (JVM) LTS version, 3.3.3, and smoke tested.   

Thank you for the suggestion. I think that using the latest Scala Long Term Support
version, unless there is a good reason not to do so, is a good convention for this template.

I also propose that we use a reasonably current Scala Native version.  Perhaps not the 
"hot bits" the day they are released, but two weeks or a month after that.  By then
any problems should have sorted out.  What do you think?